src/error.ts

```diff
@@ -16,14 +16,14 @@ const kErrorLabels = Symbol('errorLabels');
 /**
  * @internal
  * The legacy error message from the server that indicates the node is not a writable primary
- * https://github.com/mongodb/specifications/blob/b07c26dc40d04ac20349f989db531c9845fdd755/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#not-writable-primary-and-node-is-recovering
+ * https://github.com/mongodb/specifications/blob/921232976f9913cf17415b5ef937ee772e45e6ae/source/server-discovery-and-monitoring/server-discovery-and-monitoring.md#not-writable-primary-and-node-is-recovering
```

Comment:
what's our reasoning for having the commit hash on the API docs in this file but not in the other ones?

Reply:
It is to make sure the link doesn't get broken by a change to a header name. I didn't want to have any opinion on this in this PR so I had to change the commit but I didn't remove or introduce this practice. (I believe, let me know if I missed something)